src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,32 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
+    .findAll({
+      include: [
+        {
+          model: Hacker,
+          required: true,
+        },
+      ],
+    })
+    .then(applications => Promise.all(applications.map(appl => appl.hacker.getResponseStatus(appl).then(responseStatus => ({
```

Comment:
Even if they're small queries, running over 600 of them does make me a _tad_ anxious.

As for alternatives, this is just a Sequelize relation right? You can nest includes within includes to get those nested rows.

Doesn't `ApplicationResponse` belong to `HackerApplication` and not `Hacker`, as well?